pymc_experimental/tests/test_model_builder.py

```diff
@@ -24,12 +28,13 @@ class test_ModelBuilder(ModelBuilder):
     _model_type = "LinearModel"
     version = "0.1"
 
-    def build_model(self, model_config, data=None):
-
+    def build_model(self, model_instance, model_config, data=None):
+        model_instance.model_config = model_config
+        model_instance.data = data
         self.model_config = model_config
         self.data = data
 
-        with pm.Model() as self.model:
```

Comment:
I think this should stay. Ultimately we want:
1. `ModelBuilder` base-class that contains all the config dicts and data
2. `model_builder.model` which is an instance of `pm.Model()`.

Reply:
that's the outdated version. I've replaced it, but now build_model receives instance of a ModelBuilder as an argument, so it can assign the created pm.Model() directly as the instance_model.model